packages/flutter_tools/test/general.shard/drive/drive_service_test.dart

```diff
@@ -395,8 +395,9 @@ FlutterDriverService setUpDriverService({
       PrintStructuredErrorLogMethod printStructuredErrorLogMethod,
       Object compression,
       Device device,
-      Logger logger,
+      @required Logger logger,
     }) async {
+      assert(logger != null);
```

Comment:
(if it's not clear, this makes a bunch of tests fail without the change in drive_service.dart)